File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.js
function solution(num_list) {
    let oddX = '';
    let evenX = '';
    for(let i of num_list){
        i % 2 === 0 ? evenX += String(i) : oddX += String(i)
    }
    return Number(oddX) + Number(evenX)
}

File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
    oddX = [];
    evenX = [];
    for i in num_list:
        if i % 2 == 0:
            evenX.append(str(i))
        else:
            oddX.append(str(i))
    return int(''.join(evenX)) + int(''.join(oddX))